DataDefinitions/CommodityDefinition.cs

```diff
@@ -179,7 +179,7 @@ static CommodityDefinition()
                 new CommodityDefinition(128667050, 161, "EsusekuCaviar", Foods, 9625, true),
                 new CommodityDefinition(128667051, 162, "LiveHecateSeaWorms", Foods, 8737, true),
                 new CommodityDefinition(128667052, 163, "HelvetitjPearls", Foods, 10450, true),
-                new CommodityDefinition(128667053, 164, "HIPProto-Squid", Foods, 8497, true),
+                new CommodityDefinition(128667053, 164, "HIP41181Squid", Foods, 8497, true),
```

Comment:
Are we 100% certain of these changes to the EDNames?

Reply:
OK I shall trust these.

Reply:
I confirmed these changes against Howard's spreadsheet. :-)